tests/lightning_tensor/test_gates_and_expval.py

```diff
@@ -233,23 +237,23 @@ def circuit_expval():
         with pytest.raises(DeviceError):
             circuit_expval()
 
-    def test_expval_sparseH_not_supported(self):
+    def test_expval_sparseH_not_supported(self, tn_backend):
         """Test that expval of SparseH is not supported."""
         with qml.queuing.AnnotatedQueue() as q:
             qml.expval(qml.SparseHamiltonian(qml.PauliX.compute_sparse_matrix(), wires=0))
 
-        tensornet = LightningTensorNet(4, 10)
+        tensornet = LightningTensorNet(4, max_bond_dim=10, method=tn_backend)
```

Comment:
I believe it's not necessary to set up `max_bond_dim` for `tn`

Reply:
Done!